packages/slate-editor/src/modules/editor/plugins/withDeserializeHtml/withDeserializeHtml.ts

```diff
@@ -61,7 +62,18 @@ export function withDeserializeHtml(
 
                 // If there are no "nodes" then there is no interesting "text/html" in clipboard.
                 // Pass through to default "insertData" so that "text/plain" is used if available.
-                if (nodes.length > 0) {
+                if (
```

Comment:
This comment should be on top of L76 I guess?

Reply:
https://github.com/prezly/slate/pull/445/commits/9a53d507ce1d6d1942470d0df889add4f5ac6cac